src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/AppointmentService.kt

```diff
@@ -135,7 +140,7 @@ class AppointmentService(
       ?: throw EntityNotFoundException("Appointment $appointmentId does not exist")
 
     val videoLinkBooking: VideoLinkBooking? =
-      videoLinkBookingRepository.findByMainAppointmentIds(listOf(appointmentId)).firstOrNull()
+      videoLinkBookingRepository.findByAppointmentIdsAndHearingType(listOf(appointmentId), MAIN).firstOrNull()
 
     if (videoLinkBooking != null) {
       videoLinkBookingService.deleteVideoLinkBooking(appointmentId)
```

Comment:
I don't this works does it? Should that be a booking id passed in there?

Reply:
Yes